src/Facebook/Entities/AccessToken.php

```diff
@@ -178,7 +177,7 @@ public static function getAccessTokenFromCode($code, $appId = null, $appSecret =
   {
     $params = array(
       'code' => $code,
-      'redirect_uri' => '',
```

Comment:
And it's also a open door for copy paste =) I will revert this changes nevertheless.